tests/quickcheck.rs

```diff
@@ -0,0 +1,326 @@
+extern crate num_bigint;
+extern crate num_integer;
+extern crate num_traits;
+
+#[cfg(test)]
+#[cfg(feature = "quickcheck")]
+#[macro_use]
+pub extern crate quickcheck;
+
+#[cfg(test)]
+#[cfg(feature = "quickcheck")]
```

Comment:
We don't need `cfg(test)` in `tests/*.rs` files.

The feature test can be an inner attribute at the top to apply to this entire file, `#![cfg(...)]`. Then you don't need the additional `mod` at all.